sdk/cosmos/microsoft-azure-cosmos-examples/src/main/java/com/azure/data/cosmos/examples/HelloWorldDemo.java

```diff
@@ -13,43 +13,43 @@ public static void main(String[] args) {
     }
 
     void runDemo() {
-        // Create a new CosmosClient via the builder
+        // Create a new CosmosAsyncClient via the builder
         // It only requires endpoint and key, but other useful settings are available
-        CosmosClient client = CosmosClient.builder()
-            .endpoint("<YOUR ENDPOINT HERE>")
-            .key("<YOUR KEY HERE>")
-            .build();
+        CosmosAsyncClient client = CosmosAsyncClient.builder()
+            .setEndpoint("<YOUR ENDPOINT HERE>")
+            .setKey("<YOUR KEY HERE>")
+            .buildAsyncClient();
 
         // Get a reference to the container
         // This will create (or read) a database and its container.
-        CosmosContainer container = client.createDatabaseIfNotExists("contoso-travel")
+        CosmosAsyncContainer container = client.createDatabaseIfNotExists("contoso-travel")
             // TIP: Our APIs are Reactor Core based, so try to chain your calls
-            .flatMap(response -> response.database()
+            .flatMap(response -> response.getDatabase()
                     .createContainerIfNotExists("passengers", "/id"))
-            .flatMap(response -> Mono.just(response.container()))
+            .flatMap(response -> Mono.just(response.getContainer()))
             .block(); // Blocking for demo purposes (avoid doing this in production unless you must)
 
-        // Create an item
+        // Create an getItem
         container.createItem(new Passenger("carla.davis@outlook.com", "Carla Davis", "SEA", "IND"))
             .flatMap(response -> {
-                System.out.println("Created item: " + response.properties().toJson());
+                System.out.println("Created item: " + response.getProperties().toJson());
                 // Read that item 👓
-                return response.item().read();
+                return response.getItem().read();
             })
             .flatMap(response -> {
-                System.out.println("Read item: " + response.properties().toJson());
-                // Replace that item 🔁
+                System.out.println("Read getItem: " + response.getProperties().toJson());
+                // Replace that getItem 🔁
```

Comment:
Same here. I won't keep pointing it out as I find them from now.

Reply:
Corrected